frontend/javascripts/admin/api/jobs.ts

```diff
@@ -185,13 +185,11 @@ export function startNeuronInferralJob(
   datasetName: string,
   layerName: string,
   bbox: Vector6,
-  outputSegmentationLayerName: string,
```

Comment:
was there an active decision to not support providing a custom output name?

Reply:
Kinda. We made the decision to use the vx publish task in the inference workflows. Unfortunately, it does not currently support custom output layer names, and also does not link in any of the input dataset’s layers, except the color layer the inference ran on. So a standard name for the output layer also doesn’t really make sense.

If in the future we want to add these functionalities to the publish task, we can decide if we want the user to supply this name again. However, all uses we saw used the suggested standard name, so maybe it’s not worth the extra complexity for the user anyway.

Reply:
> Unfortunately, it does not currently support custom output layer names, and also does not link in any of the input dataset’s layers, except the color layer the inference ran on

hm, ok, it feels like a step back, but I understand that it doesn't make sense now due to the publish task. let's see when it will become a pain point again. luckily, we can compose datasets in the UI afterwards now.